jsign-crypto/src/main/java/net/jsign/jca/AmazonSigningService.java

```diff
@@ -81,6 +81,21 @@ public class AmazonSigningService implements SigningService {
         algorithmMapping.put("SHA512withRSA/PSS", "RSASSA_PSS_SHA_512");
     }
 
+     /**
+     * Generates the endpoint URL for the given AWS region.
+     *
+     * @param region the AWS region
+     * @return the endpoint URL
+     */
+    public static String getEndpointUrl(String region) {
```

Comment:
Is it possible to make this method non static and package privarte?

Reply:
I assumed, if I want to test this individually, it needs to be public. Don't know that much about Java ;) Can give it a try later …

Reply:
It remains testable from a test class in the same package if the public keyword is removed, that's the "package private" visibility.